llvm/lib/Analysis/IR2Vec.cpp

```diff
@@ -55,6 +55,40 @@ static cl::opt<float> ArgWeight("ir2vec-arg-weight", cl::Optional,
 
 AnalysisKey IR2VecVocabAnalysis::Key;
 
+// ==----------------------------------------------------------------------===//
+// Embedding
+//===----------------------------------------------------------------------===//
+
+Embedding &Embedding::operator+=(const Embedding &RHS) {
+  assert(this->size() == RHS.size() && "Vectors must have the same dimension");
+  std::transform(this->begin(), this->end(), RHS.begin(), this->begin(),
```

Comment:
was going to suggest `llvm::transform` but looks like there's no overload for this

Reply:
Yeah! And, `Embedding` is built on std::vector, so thought of just using std::transform.